libraries/chain/include/cyberway/genesis/ee_genesis_container.hpp

```diff
@@ -1,22 +1,31 @@
 #pragma once
 
 #include <eosio/chain/types.hpp>
+#include <eosio/chain/abi_def.hpp>
 #include <fc/reflect/reflect.hpp>
 
 namespace cyberway { namespace genesis {
 
 using namespace eosio::chain;
 
-struct genesis_ee_header {
-    char magic[14] = "CyberwayEEGen";
+struct ee_genesis_header {
+    char magic[12] = "CyberwayEEG";
     uint32_t version = 1;
+    fc::sha256 hash;
 
     bool is_valid() {
-        genesis_ee_header oth;
+        ee_genesis_header oth;
         return string(magic) == oth.magic && version == oth.version;
     }
 };
 
+struct ee_table_header {
```

Comment:
I already have section_ee_header. For what two headers? Make your changes to my.

Reply:
section_ee_header too hardly coded types of genesis structure. Our goal to make Event Engine genesis more flexible, so another application can use this function.

Reply:
@s-medvedev OK, I agree, but remove section_ee_header and rewrite all usages.

Reply:
I think, this changes should be done in another issue. How do the changes you indicated relate to this task?

Reply:
See issue #559